public/app/features/explore/Wrapper.tsx

```diff
@@ -67,13 +82,14 @@ class WrapperUnconnected extends PureComponent<Props> {
   }
 
   render() {
+    const styles = getStyles();
     const { left, right } = this.props.queryParams;
     const hasSplit = Boolean(left) && Boolean(right);
 
     return (
-      <div className="page-scrollbar-wrapper">
+      <div className={cx(styles.pageScrollbarWrapper)}>
         <ExploreActions exploreIdLeft={ExploreId.left} exploreIdRight={ExploreId.right} />
-        <div className="explore-wrapper">
+        <div className={cx(styles.exploreWrapper)}>
```

Comment:
you don't need `cx` if you are not applying classnames conditionally:
```suggestion
        <div className={styles.exploreWrapper}>
```